tools/mtouch/Application.cs

```diff
@@ -1836,6 +1845,37 @@ protected override void Build ()
 		}
 	}
 
+	public class MsymCopyTask : ProcessTask {
+		
+		public string Source { get; set; }
+		public string Destination { get; set; }
+		
+		public MsymCopyTask (string source, string destination) {
+			Source = source;
+			Destination = destination;
+			ProcessStartInfo.FileName = "mono-symbolicate";
+			ProcessStartInfo.Arguments = $"store-symbols \"{Source}\" \"{Destination}\"";
+		}
+		
+		protected override void Build ()
+		{
+			if (String.IsNullOrEmpty (Source)) {
+				Console.Error.WriteLine ("Symbolcation error, could not copy msym files because the source directory is missing.");
+				throw new ArgumentNullException (nameof (Source));
```

Comment:
not ideal - but it should never happen

Unhanded exceptions becomes `MT0000` errors that ask customers to file bug reports. This is something (if it could be `null` that would require it's own error message as it would likely be fixable - without  requiring a bug report).